executor/distsql.go

```diff
@@ -213,6 +213,7 @@ func rebuildIndexRanges(ctx sessionctx.Context, is *plannercore.PhysicalIndexSca
 type IndexReaderExecutor struct {
 	baseExecutor
 
+	// Table is a PhysicalTable for partitioned table.
```

Comment:
This comment looks weird, can we elaborate it more or just remove it?

Reply:
Updated, is it better now?